index.html

```diff
@@ -0,0 +1,114 @@
+    <link rel="stylesheet" href="css/index.css">
+    <script src="js/index.js" type="module" async></script>
+    <title>Creative Commons Global Summit 2015</title>
+  </head>
+  <body>
+    <main id="main" class="container-fullwidth">
+      <header>
+        <nav id="nav" class="container-fullwidth navbar">
+          <div class="desktop-visible" id="top-nav" >
+            <a href="https://facebook.com"><img src="assets/icons/facebook.png" alt="Facebook logo"></a>
+            <a href="https://twitter.com"><img src="assets/icons/twitter.png" alt="Twitter logo"></a>
+            <a href="#">Logout</a>
+            <a href="#">Mypage</a>
+            <a href="#">English</a>
+          </div>
+          <img src="assets/icons/mobile-menu.png" alt="Menu hamburger button" id="hamburger" class="nav-item mobile-visible">
+          <div class="nav-options row-fullwidth mobile-visible" id="bottom-nav">
+            <a class="desktop-visible" id="logo" href="index.html"><img src="assets/images/logo-clickable.png" alt="Clickable logo"></a>
+            <img class="mobile-visible close-menu" src="assets/icons/close-icon.png" alt="Close menu button.">
+            <div class="mobile-menu">
+              <a class="nav-option current-page" href="html/about.html">About</a>
+              <a class="nav-option" href="#programs">Program</a>
+              <a class="nav-option mobile-visible" href="#join-summit">Join</a>
+              <a class="nav-option desktop-visible" href="#seewhole">Join</a>
+              <a class="nav-option" href="#sponsors">Sponsor</a>
+              <a class="nav-option" href="https://creativecommons.org/tag/news/">News</a>
+              <a class="nav-option" href="https://creativecommons.org/campaign/">CC Campaign</a>
+            </div>
+          </div>
+        </nav>
+      </header>
+      <section id="event-intro" class="container-fullwidth">
+        <span class="pre-title">"Hello! Sharing World"</span>
+        <h1 class="title">Creative Commons Global Summit 2015</h1>
+        <p class="introduction">
+          A joyful celebration believing in the value of openness and 
+          sharing, creating a positive change with people from over 80 
+          countries is taking place in October, in Korea.
+        </p>
+        <article class="datetime-place">
+          <h2>2015.10.15(THU) ~ 16(FRI)</h2>
+          <div>
+            @ National Museum of Korea, Art Center Nabi and more
+          </div>
+        </article>
+      </section>
+    </main>
+    <section class="container-fullwidth" id="programs">
+      <h2>Main Program</h2>
+      <hr class="hr">
+      <ul id="programs-list">
+        <li id="program1" class="program-item" tabindex="0">
+          <img src="assets/icons/program1.png" alt="Lecture Icon">
```

Comment:
- [ ] When we hover the mouse over cards a white border around cards should appear as on the design, Please apply hover effects to your cards so the cards show a white border when we hover the mouse over them.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/157689981-91c089cf-d233-49a4-b0d5-3e74f7a6d17b.png)